chainerx_cc/chainerx/native/native_device/linalg.cc

```diff
@@ -0,0 +1,188 @@
+#include "chainerx/native/native_device.h"
+
+#include <cmath>
+#include <cstdint>
+#include <type_traits>
+
+#include "chainerx/array.h"
+#include "chainerx/backend.h"
+#include "chainerx/device.h"
+#include "chainerx/dtype.h"
+#include "chainerx/indexable_array.h"
+#include "chainerx/kernels/creation.h"
+#include "chainerx/kernels/linalg.h"
+#include "chainerx/macro.h"
+#include "chainerx/native/data_type.h"
+#include "chainerx/native/elementwise.h"
+#include "chainerx/native/kernel_regist.h"
+#include "chainerx/routines/creation.h"
+#include "chainerx/shape.h"
+
+#ifdef CHAINERX_ENABLE_LAPACK
+// gesv
+extern "C" void dgesv_(int* n, int* nrhs, double* a, int* lda, int* ipiv, double* b, int* ldb, int* info);
+extern "C" void sgesv_(int* n, int* nrhs, float* a, int* lda, int* ipiv, float* b, int* ldb, int* info);
+
+// getrf
+extern "C" void dgetrf_(int* m, int* n, double* a, int* lda, int* ipiv, int* info);
+extern "C" void sgetrf_(int* m, int* n, float* a, int* lda, int* ipiv, int* info);
+
+// getri
+extern "C" void dgetri_(int* n, double* a, int* lda, int* ipiv, double* work, int* lwork, int* info);
+extern "C" void sgetri_(int* n, float* a, int* lda, int* ipiv, float* work, int* lwork, int* info);
+#endif
```

Comment:
Could you append a comment for readability and per convention with the variable? Ditto for other similar occurrences incl. `#else` lines.
```suggestion
#endif // CHAINERX_ENABLE_LAPACK
```